azurelinuxagent/common/osutil/clearlinux.py

```diff
@@ -42,6 +42,14 @@ def __init__(self):
         self.agent_conf_file_path = '/usr/share/defaults/waagent/waagent.conf'
         self.jit_enabled = True
 
+    @staticmethod
+    def get_systemd_unit_file_install_path():
+        return "/usr/lib/systemd/system"
+
+    @staticmethod
+    def get_agent_bin_path():
+        return "/usr/bin"
```

Comment:
we need to ensure those values are consistent with the values in setup.py (maybe setup should use osutil???)

Reply:
done already, cool